.github/scripts/create_env_file.sh

```diff
@@ -16,6 +16,9 @@ DREMIO_DATABASE=dbt_test
 DBT_TEST_USER_1=dbt_test_user_1
 DBT_TEST_USER_2=dbt_test_user_2
 DBT_TEST_USER_3=dbt_test_user_3
+DBT_TEST_ROLE_1=dbt_test_role_1
+DBT_TEST_ROLE_2=dbt_test_role_2
+DREMIO_EDITION=community
```

Comment:
We should describe it in the docs of the tests at least. And create an enum with possible values, which may not be in this PR.